### PR TITLE
Add notebook to use for review live demo

### PR DESCRIPTION
**Pull request by sjspielman**
Towards #5 

This PR does a couple interesting things to get ready for the analytical code review live demo, such that I hope it makes next prep stages go more smoothly!(??)

- I modified the directory name `.live-demo-issues` -> `.live-demo-content` to better reflect contents, and added a README.
- I updated the `scripts/explore-spotify-variation.Rmd` that can be filed for the "leaving an analytical code review" live demo, via several commits with commit messages at varying levels of detail. 
  - Turns out I unintentionally have a commit message typo in there which I did not notice (I typed `asn` instead of `and` - let's call it realistic 😂 )
  - There are _four commits_ that do this file modification: `665acb4`...`56f4937`
- Then, I copied this final notebook into `.live-demo-content/explore-spotify-variation-with-UMAPS.md`
- I have **not yet** taken this next step to wrap up the PR; in case we want more changes to the Rmd, I figured let's discuss before I take this not-entirely-destructive-but-sorta-destructive action. Next up in this PR, I will `git revert` those 4 commits. This way, the updated notebook in `scripts/` does not get merged in, but the "template" of this does get merged into `.live-demo-content/`. Further, the commits that should be in the PR for the live demo actually now exist in the repo history.
  - Then, to actually set up the branch ready for filing the PR for the live demo, you can cherry-pick those 4 commits (`665acb4`...`56f4937`) into the branch to file, and again, they live on in project history. 

Here's the rendered notebook: [explore-spotify-variation.nb.html.zip](https://github.com/AlexsLemonade/2023-chop-training-demo/files/12217413/explore-spotify-variation.nb.html.zip)

Some additional things along the way:
- I ended up using `{patchwork}`. To emphasize the planned review comment (see #5) about needing `renv`, I will plan to _not_ have `{patchwork}` installed. This way we have two funky packages situations: `{umap}` is not in the environment but I have installed , _and_ `{patchwork}` is in the environment but I don't it installed. --> bonus reproducibility teachable moments 
- I accidentally committed a bug, but decided to leave it so that it can also be reviewed - this line is missing a type of label (should be `labs(title = "..")`) - https://github.com/AlexsLemonade/2023-chop-training-demo/blob/f8150cfb394c9be177b98a46f910c581c4fa963a/.live-demo-content/explore-spotify-variation-with-UMAPs.Rmd#L119
  - This plots legend title is also not fantastic, but it's not _bad_ if it's for internal-only code that we don't expect others to necessarily consume, so this leaves a chance to discuss how intense one needs to be for reviews.

### CODE REVIEW

File: .live-demo-content/git-basics.md
Comment (jaclyn-taroni) [inline .live-demo-content/git-basics.md:1]: I would put this in a subdirectory that makes it obvious this is an issue template.

File: scripts/explore-spotify-variation.Rmd
Comment (jaclyn-taroni) [inline scripts/explore-spotify-variation.Rmd:1]: Just realized this is in `scripts/` – maybe we'd be better off calling this folder `analyses`?

**Review by jaclyn-taroni — COMMENTED** [review]
Looks good overall. The comments I am returning are organizational, but could cause problems with the cherry-picking plans.

File: scripts/explore-spotify-variation.Rmd
Comment (sjspielman) [reply scripts/explore-spotify-variation.Rmd:1]: :100:

File: .live-demo-content/git-basics.md
Comment (sjspielman) [reply .live-demo-content/git-basics.md:1]: done in https://github.com/AlexsLemonade/2023-chop-training-demo/pull/14/commits/3fe50849746505a734568a689f1015a5d1ed08dc & https://github.com/AlexsLemonade/2023-chop-training-demo/pull/14/commits/4ea69b97b4f3e78da8b9e18b3956b2729b886a05

File: scripts/explore-spotify-variation.Rmd
Comment (sjspielman) [reply scripts/explore-spotify-variation.Rmd:1]: See https://github.com/AlexsLemonade/2023-chop-training-demo/pull/14/commits/7e17c45bbf0dcfd94763fdf70e205b5bae67b435 and https://github.com/AlexsLemonade/2023-chop-training-demo/pull/14/commits/66544bf994510464df14fb300ecf6d57d12d7294

**Comment by sjspielman** [comment]
Unclear how cherry picking will go with `scripts` -> `analyses`, but at least the majority of the work is very templated out if you have to "redo" it!

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻

**Pull request merged**